pkg/utils/deleterelayservice.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func DeleteRelayAgent(kubeConfig []byte, namespace string) bool {
+
+	kubeconfig := flag.String("kubeconfig", string(kubeConfig[:]), "kubeconfig file yaml byte")
+	flag.Parse()
+
+	config, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
+	if err != nil {
+		fmt.Println("Unable to build kube configuration ", err.Error())
+		config, err = rest.InClusterConfig()
```

Comment:
this also should not be present as InClusterConfig picks up the config where paralus controller is running however we want to delete relay-agent from the target cluster